#goal definition
geometry_msgs/Pose2D robot_goal
---
#result definition
bool reached_goal
---
#feedback
geometry_msgs/Pose2D actual_pose
